contracts/contracts/GrantRegistry.sol

```diff
@@ -0,0 +1,135 @@
+pragma solidity ^0.8.5;
+
+contract GrantRegistry {
+  // Grant ID - The hash of the grants meetadata
```

Comment:
I'm going to pushback on using the hash of the metadata as the grant ID, for the reasons @apbendi has previously mentioned—an onchain transaction to deploy a new grant just to fix a typo is terrible (and expensive) UX

Reply:
makes sense but since this lives beyond anyone's control we don't want weird scenario like me getting my grant verified by the DAO and then updating content to pose as metamask / umbra / etc. Just a thought